app/scripts/controllers/app-state.js

```diff
@@ -288,10 +288,19 @@ export default class AppStateController extends EventEmitter {
       return;
     }
 
+    // This is a temporary fix until we add a state migration.
+    // Due to a bug in ui/pages/settings/advanced-tab/advanced-tab.component.js,
+    // it was possible for timeoutMinutes to be saved as a string, as explained
+    // in PR 25109. `alarms.create` will fail in that case. We are
+    // converting this to a number here to prevent that failure. Once
+    // we add a migration to update the malformed state to the right type,
+    // we will remove this conversion.
+    const timeoutToSet = Number(timeoutMinutes);
```

Comment:
Do we have an issue # for the migration task? I want to add a comment to it that this cast should be removed when the migration work is done.

Reply:
I do not think we have created one yet